src/Akka.Remote.Hosting/AkkaRemoteHostingExtensions.cs

```diff
@@ -8,6 +8,11 @@ public static class AkkaRemoteHostingExtensions
     {
         private static AkkaConfigurationBuilder BuildRemoteHocon(this AkkaConfigurationBuilder builder, string hostname, int port, string publicHostname = null, int? publicPort = null)
         {
+            if (string.IsNullOrEmpty(publicHostname))
```

Comment:
Decided to resolve a potential papercut for most users here - have the system bind, by default, to a reachable socket and alias using the hostname if a `publicHostname` isn't explicitly specified. Means that most K8s / Docker scenarios should "just work" with these changes.